content/en/security/cloud_security_management/setup/windows.md

```diff
@@ -0,0 +1,89 @@
+---
+title: Setting Up Cloud Security Management On Windows
+kind: documentation
+is_beta: true
+private: true
+---
+
+<div class="alert alert-warning">Cloud Security Management on Windows is in private beta and should only be installed on hosts that are not critical to production workloads.</div>
+
+Datadog [Cloud Security Management (CSM)][1] on Windows includes built-in threat detection for Windows process and network events. The out-of-the-box Windows ruleset includes the following default rules:
+
+- Certutil used to transmit or decode a file
+- Process memory was dumped using the minidump functions of comsvcs.dll
+- NTDS file referenced in commandline
```

Comment:
```suggestion
- NTDS file referenced in command line
```

Reply:
Good catch. This is actually a rule name, so I'm going to leave as-is and alert the PM that we should update the rule name (and then I'll circle back and update this in the doc).